android/build.gradle

```diff
@@ -15,6 +25,10 @@ android {
     }
 }
 
+repositories {
+    mavenCentral()
+}
+
```

Comment:
are these just changes required by latest RN?

Reply:
@sarmadka I'm not entirely sure but I needed to do this in order to get gradle project sync to work in android studio so I could reproduce the error without having to reinstall. I modeled it off of the one from https://github.com/frostney/react-native-create-library . I figured I may as well commit as it doesn't seem to have that much of an effect either way.